go.mod

```diff
@@ -7,16 +7,18 @@ require (
 	github.com/ampproject/amphtml v0.0.0-20180912232012-d3df64d07ae9
 	github.com/go-acme/lego/v3 v3.2.0
 	github.com/gofrs/flock v0.7.1
-	github.com/golang/protobuf v1.3.2
-	github.com/google/go-cmp v0.3.0
+	github.com/golang/protobuf v1.4.1
+	github.com/google/go-cmp v0.5.0
+	github.com/kylelemons/godebug v1.1.0
 	github.com/pelletier/go-toml v1.1.0
 	github.com/pkg/errors v0.8.1
 	github.com/pquerna/cachecontrol v0.0.0-20180306154005-525d0eb5f91d
 	github.com/prometheus/client_golang v1.1.0
 	github.com/stretchr/testify v1.4.0
 	golang.org/x/crypto v0.0.0-20190701094942-4def268fd1a4
 	golang.org/x/net v0.0.0-20190930134127-c5a3c61f89f3
-	google.golang.org/grpc v1.20.1
+	google.golang.org/grpc v1.27.0
```

Comment:
Just to double check, did you try building the amppkg with -tags dns01? Just to make sure these library upgrades didn't break that feature.

Reply:
Yes, it builds. Though, when running I get this on stdout:

```
WARNING: Package "github.com/golang/protobuf/protoc-gen-go/generator" is deprecated.
        A future release of golang/protobuf will delete this package,
        which has long been excluded from the compatibility promise.
```

Reply:
Filed #482. I don't think it's a release blocker.